donate/templates/pages/core/campaign_page_master.html

```diff
@@ -50,21 +50,6 @@ <h1 class="heading heading--primary hero__heading">{{ page.title }}</h1>
                         {{ page.intro|richtext }}
                     </div>
                 </div>
-                {% block logo_showcase %}
-                <div class="logo-showcase column-spacing">
-                    <div class="logo-showcase__container">
-                        <div class="logo-showcase__container">
-                            <div class="logo-showcase__item logo-item">
-                                <a href="https://www.charitynavigator.org/index.cfm?bay=search.summary&orgid=16300" target="_blank"><img class="logo-item__image" src="{% static '_images/seal-charity-navigator-4-star.png' %}" alt="{% trans 'Seal of Charity Navigator 4-Star' %}"></a>
-                            </div>
-                            <div class="logo-showcase__item logo-item">
-                                <a href="https://www.guidestar.org/profile/20-0097189" target="_blank"><img class="logo-item__image" src="{% static '_images/seal-guidestar-gold-2019.png' %}" alt="{% trans 'Seal of GuideStar Gold 2019' %}"></a>
```

Comment:
Let's remove the image file `seal-guidestar-gold-2019.png`.

Reply:
Removed!